src/engine/build.sh

```diff
@@ -77,6 +77,7 @@ Guess_Toolset ()
           BOOST_JAM_TOOLSET=vacpp
        fi
     elif test_uname AIX && test_path xlc; then BOOST_JAM_TOOLSET=vacpp    
+    elif test_uname FreeBSD && test_path freebsd-version; then BOOST_JAM_TOOLSET=clang
```

Comment:
AMDG

On 02/29/2016 08:37 AM, Rene Rivera wrote:
>> @@ -77,6 +77,7 @@ Guess_Toolset ()
>>            BOOST_JAM_TOOLSET=vacpp
>>         fi
>>      elif test_uname AIX && test_path xlc; then BOOST_JAM_TOOLSET=vacpp    
>> +    elif test_uname FreeBSD && test_path freebsd-version; then BOOST_JAM_TOOLSET=clang
> 
> Is this is for FreeBSD >=10.. Shouldn't the version also be checked?
> 

  Also, shouldn't this be testing whether clang exists?

In Christ,
Steven Watanabe